src/config.ts

```diff
@@ -194,6 +194,8 @@ const ONBOARDING_FEATURES_ALL_DISABLED = Object.fromEntries(
   Object.values(ToggleableOnboardingFeatures).map((value) => [value, false])
 )
 
+export const CYA_ENABLED = stringToBoolean(Config.CYA_ENABLED || 'false')
```

Comment:
Wouldn't this turn CYA off? Is this set to true somewhere?

Reply:
We can actually remove CYA completely; updated this PR to account for that